webpack.production.config.js

```diff
@@ -0,0 +1,25 @@
+var webpack = require('webpack');
+var path = require('path');
+
+var BUILD_DIR = path.resolve(__dirname, 'client/dist');
+var APP_DIR = path.resolve(__dirname, 'client/src');
+
+var config = {
+  entry: './client/src/productionView.jsx',
```

Comment:
config points to `productionView.jsx` but the filename is `productionView.js`, could be problematic?  You may also want to add a package.json script to point webpack to this production config file.  Otherwise, looks good!